package.json

```diff
@@ -24,22 +24,24 @@
         "cueball": "2.2.9",
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
-        "dtrace-provider": "0.2.8",
+        "dtrace-provider": "0.8.8",
         "fast": "2.8.1",
         "http-signature": "1.1.0",
         "jsprim": "^1.3.1",
         "kang": "1.1.0",
         "keep-alive-agent": "0.0.1",
         "keyapi": "git+https://github.com/joyent/keyapi.git#e14b3d58",
         "libmanta": "git+https://github.com/joyent/node-libmanta.git#v1.3.0",
-        "libuuid": "0.1.2",
+        "uuid": "3.3.3",
         "lstream": "0.0.4",
         "mahi": "2.3.1",
-        "mime": "1.2.11",
+        "mime": "1.4.0",
         "moray": "3.4.1",
         "once": "1.3.0",
-        "restify": "2.6.3",
-        "sshpk": "1.13.1",
+        "restify": "6.4.0",
+        "restify-clients": "2.6.7",
+        "restify-errors": "7.0.0",
```

Comment:
Hopefully not, but there is potential fallout from this new restify-errors.
Note that restify v6 uses restify-errors v5. Why using v7 here? It might be
fine.

Reply:
good catch.  I've updated errors to v5 to match what restify itself is using.